src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/SelectDeploymentType.tsx

```diff
@@ -69,27 +75,88 @@ const CardList = ({ items, isReadonly, selectedValue, onChange, allowDisabledIte
         )
         return (
           <Utils.WrapOptionalTooltip key={item.value} tooltipProps={item.tooltipProps} tooltip={item.tooltip}>
-            {allowDisabledItemClick ? <div onClick={() => onChange(item.value)}>{itemContent}</div> : itemContent}
+            {allowDisabledItemClick ? (
+              <div onClick={() => onChange(item.value as ServiceDeploymentType)}>{itemContent}</div>
+            ) : (
+              itemContent
+            )}
           </Utils.WrapOptionalTooltip>
         )
       })}
     </Layout.Horizontal>
   )
 }
 
+const getCGTypes = (
+  cgSupportedDeploymentTypes: DeploymentTypeItem[],
+  nativeHelmFF = false,
+  serverlessFF = false
+): DeploymentTypeItem[] => {
+  let cgTypes = cgSupportedDeploymentTypes
+  if (nativeHelmFF) {
+    cgTypes = cgTypes.filter(deploymentType => deploymentType.value !== 'NativeHelm')
+  }
+  if (serverlessFF) {
+    cgTypes = cgTypes.filter(deploymentType => !isServerlessDeploymentType(deploymentType.value))
+  }
+  return cgTypes
+}
+
+const getServerlessDeploymentTypes = (
+  getString: (key: keyof StringsMap, vars?: Record<string, any> | undefined) => string,
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232297)
```suggestion
  getString: (key: keyof StringsMap, vars?: Record<string, unknown> | undefined) => string,
```